<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Klassenteilungen</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Divisionen</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="divisions">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <item>
          <property name="text">
           <string>A+B</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G+R</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edit_division"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="remove_division">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../icons/icon_remove.svg</normaloff>../icons/icon_remove.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="new_division">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../icons/icon_new.svg</normaloff>../icons/icon_new.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>überflüssige Untergruppen</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="redundant_subgroups">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Einträge der Form „Untergruppe=Gruppe“, z.B. „A.P=P“ oder „A.M.R=A.M“, geben an, dass die Untergruppe identisch ist mit der Gruppe auf der rechten Seite.&lt;/p&gt;&lt;p&gt;So kann die Anzahl der Elementargruppen reduziert werden.&lt;/p&gt;&lt;p&gt;Die Grundgruppen der rechten Seite müssen (typografisch) eine Untermenge der Grundgruppen der linken Seite bilden, z.B. {A, M} ⊂ {A, M, R}. Das ist natürlich das Gegenteil der normalen Beziehung zwischen den Schülergruppen: A.M.R ⊂ A.M.&lt;/p&gt;&lt;p&gt;Die Aussage der Angabe „A.M.R=A.M“ ist allerdings, dass die Mengen identisch sind.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <item>
          <property name="text">
           <string>A.G=A</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B.R=R</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edit_equality"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="remove_equality">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../icons/icon_remove.svg</normaloff>../icons/icon_remove.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="new_equality">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../icons/icon_new.svg</normaloff>../icons/icon_new.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Elementargruppen</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="atomic_groups">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <item>
          <property name="text">
           <string>A</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B.G</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>R</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="subgroups">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>60</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Gruppe</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Untergruppen</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>B</string>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string>B.G+R</string>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>G</string>
          </property>
         </item>
         <item row="1" column="1">
          <property name="text">
           <string>A+B.G</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Bericht:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="analysis">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>divisions</tabstop>
  <tabstop>edit_division</tabstop>
  <tabstop>remove_division</tabstop>
  <tabstop>new_division</tabstop>
  <tabstop>redundant_subgroups</tabstop>
  <tabstop>edit_equality</tabstop>
  <tabstop>remove_equality</tabstop>
  <tabstop>new_equality</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>divisions</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>edit_division</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
